dev_docs/tutorials/development_windows.mdx

```diff
@@ -0,0 +1,78 @@
+---
+id: kibDevTutorialSetupWindowsDevWSL
+slug: /kibana-dev-docs/tutorial/setup-windows-development-wsl
+title: WSL on Windows Development
+summary: Learn how to setup a Windows development environment using WSL
+date: 2022-06-16
+tags: ['kibana', 'onboarding', 'setup', 'windows', 'development', 'wsl']
+---
+
+
+# Overview
+
+Development on Windows is recommended through WSL2.  WSL lets users run a Linux environment on Windows, providing a supported development environment for Kibana.
+
+## Install WSL
+
+<DocCallOut title="Prerequisites">
+  You must be running Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11.
+</DocCallOut>
+
+The latest setup instructions can be found at https://docs.microsoft.com/en-us/windows/wsl/install
+
+1) Open Powershell as an administrator
+
+2) Install WSL with Ubuntu and restart the machine when prompted
+    ```
+    wsl --install -d Ubuntu
+    ```
+
+    In case you already have WSL installed you can just `wsl --update` and then reboot the machine.
+
+3) Once rebooted the installation will proceed for a new WSL install. If that's not the case make sure you're using 
+   WSL2 and Ubuntu by
+   ```
+   wsl --set-default-version 2
+   ```
+
+   and
+
+   ```
+   wsl --set-default Ubuntu 
+   ```  
+
+4) Install support for Google Chrome by running the following inside WSL shell
+
+   ```
+   sudo apt update && sudo apt -y upgrade && sudo apt -y autoremove 
+   cd /tmp
+   sudo wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+   sudo dpkg -i google-chrome-stable_current_amd64.deb
+   sudo apt install --fix-broken -y
+   sudo dpkg -i google-chrome-stable_current_amd64.deb
+   ```
+
+  Now you can launch Google Chrome by either `google-chrome` on your WSL shell or from Ubuntu Applications on Windows. 
```

Comment:
No luck on Windows 10 unfortunately, the link below indicates it's only Windows 11+

Reply:
Nice catch! I've updated the guide to accommodate that

Reply:
@jbudz I believe we can have that split between basic and default experiences for now, and then in a couple months fully require Windows 11. What do you think about it?

Reply:
:+1: on the split